### DISCUSSION TITLE
image keeps blinking

### DISCUSSION THREAD

**Discussion by d0razi** [cotes2020/jekyll-theme-chirpy#1180] [Q&A]
https://d0razi.github.io/

When I go to the above site, the image is completely loaded, but it keeps blinking, so how do I solve it? Can  i get that blinking effect out of the source code?

**Comment by JinchaoLove** [comment]
Not blinking in latest Chrome and Safari, what's the browser you used?
<img width="1410" alt="image" src="https://github.com/cotes2020/jekyll-theme-chirpy/assets/34153355/782c9228-2518-406f-a83f-16cc2d502776">

**Comment by d0razi** [comment]
It just happened to be solved. I think it was solved by adding the toc:false option.

**Reply by d0razi** [reply]
This option was completely irrelevant. The problem was the Google addguard extension.

**Comment by ciphermenial** [comment]
This is fixed in the latest version.

**Comment by jefflongo** [comment]
I don't think this is fixed? If I remove this line from the current master branch
https://github.com/cotes2020/jekyll-theme-chirpy/blob/37c976499ead51c1d88e8e8213366240a72adebc/_posts/2019-08-08-text-and-typography.md?plain=1#L12
it will shimmer forever.

EDIT: after deploying it seems to work, so it must be some sort issue with my local environment setup?

**Comment by ciphermenial** [comment]
> EDIT: after deploying it seems to work, so it must be some sort issue with my local environment setup?

That is a possibility. After updating to the latest version on my github pages, the shimmering stopped.

**Comment by hojourney** [comment]
My blog(https://hojourney.github.io/) has the same issue. Can anybody help me solve this problem?
<img width="731" alt="Screenshot 2023-10-02 at 15 53 05" src="https://github.com/cotes2020/jekyll-theme-chirpy/assets/93585100/738a4fa4-7f59-498e-ac76-b3953449760f">

**Comment by ciphermenial** [comment]
> My blog(https://hojourney.github.io/) has the same issue. Can anybody help me solve this problem?

Yes, update.
